vision/image_simulator/create_data/test_labels.p

```diff
@@ -0,0 +1,150 @@
+(lp1
```

Comment:
It looks like there's a script to generate this file, so don't check in the raw pickle output. Add a line to .gitignore to stop git from tracking this file:
vision/image_simulator/create_data/test_labels.p
and also make sure that you git rm this file (and the other generated files that don't have code in them)